lib/fluent/config/dsl.rb

```diff
@@ -27,10 +28,55 @@ def self.read(path)
         end
 
         def self.parse(source, source_path="config.rb")
+          confpath = File.dirname(File.expand_path(source_path))
+          endres = parse_include(source,confpath,[])
```

Comment:
Please put whitespace between each arguments like below.
`parse_include(source, confpath, [])`

Please apply to other functions.